includes/google_scholar.inc

```diff
@@ -171,7 +179,7 @@ function islandora_scholar_create_meta_tags($object) {
 
     $online_date = $mods_xml->xpath(variable_get('islandora_scholar_xpaths_online_date', '//mods:recordInfo/mods:recordCreationDate'));
     if ($online_date) {
-      $date_string = islandora_scholar_parse_date_foryear($online_date);
+      $date_string = islandora_scholar_date_foryear($online_date);
```

Comment:
We originally had `islandora_scholar_parse_date_foryear` which was correct but the original commits changed this to `islandora_parse_date_foryear` leaving out the `_scholar_` bit. Now it appears that you have added `_scholar_` but in the process left out the `_parse_` part. This needs to be `islandora_scholar_parse_date_foryear`, anything else causes the object page to fail to load. Please test a page load for the object before committing to verify that the object page loads.